crates/polars-ops/src/series/ops/unique.rs

```diff
@@ -39,6 +39,9 @@ pub fn unique_counts(s: &Series) -> PolarsResult<Series> {
             DataType::String => {
                 Ok(unique_counts_helper(s.str().unwrap().into_iter()).into_series())
             },
+            DataType::Boolean => {
+                Ok(unique_counts_helper(s.bool().unwrap().into_iter()).into_series())
```

Comment:
I think we should at least add a few fast paths. We know the amount of set bits of both the validity and the underlying boolean bitmasks.

Reply:
I made some changes, but must admit I'm on deep water here. Would love some feedback if this even in the right direction. If not I guess I'd need some more guidance in order to be able to assist here.

Reply:
@ritchie46 There have been quite some refactorings since you last had a look on this, and now there are fast paths for all combinations of true/false/null as far as I'm concerned. Please have a look when you find the time to see if this is in the right direction, or if maybe someone else should have a go at it instead.